### ISSUE TITLE
Incompatible with TrajectoryOptimization

### ISSUE THREAD

**Issue by kunalmenda** [bjack205/RobotZoo.jl#1]
Hey!

Just tried adding TrajectoryOptimization, building TrajectoryOptimization, and then adding RobotZoo. I ran into conflicting requirements:

`(v1.3) pkg> add RobotZoo
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Rotations [6038ab10]:
 Rotations [6038ab10] log:
 ├─possible versions are: [0.7.2, 0.8.0, 0.9.0-0.9.2, 0.10.0, 0.11.0-0.11.1, 0.12.0-0.12.1, 0.13.0, 1.0.0-1.0.1] or uninstalled
 ├─restricted by compatibility requirements with TrajectoryOptimization [c79d492b] to versions: 0.13.0
 │ └─TrajectoryOptimization [c79d492b] log:
 │   ├─possible versions are: [0.0.1, 0.1.0-0.1.2, 0.2.0] or uninstalled
 │   └─restricted to versions 0.2.0 by an explicit requirement, leaving only versions 0.2.0
 └─restricted by compatibility requirements with RobotZoo [74be38bb] to versions: 1.0.0-1.0.1 — no versions left
   └─RobotZoo [74be38bb] log:
     ├─possible versions are: 0.1.0 or uninstalled
     └─restricted to versions * by an explicit requirement, leaving only versions 0.1.0`

I'm running Julia 1.3 on Ubuntu 16.04.
Let me know if I can provide any additional information. Thanks!

**Comment by bjack205** [comment]
Yes, not all the latest versions are fully registered yet. I just released the newest version of TrajOpt (0.3.0) this morning. RobotDynamics should be good to go, and ALTRO should be registered soon.
